paper/paper.md

```diff
@@ -309,26 +324,20 @@ This shows how Popper helps improve the performance of scientific workflows dras
 
```

Comment:
this paragraph is a bit big, we can break it down into 2-3

Reply:
also, the headings should use `##` rather than `###`

Reply:
Fixed the headings. Can you tell which paragraph to break ? Did you mean the kubernetes case study para ?

Reply:
actually k8s, slurm and maybe ci as well. In general a paragraph is self-contained in terms of what it tries to communicate. So for example ci might read better as one paragraph